src/__tests__/stubs/moby-epub3-subdir-navdoc.ts

```diff
@@ -14,13 +14,15 @@ const mobyEpub3Manifest = {
       type: 'application/xhtml+xml',
       href: 'OPS/subdir/1902/chapter_001.xhtml',
     },
+    {
+      type: 'application/xhtml+xml',
+      href: 'OPS/subdir/1902/toc.xhtml',
+    },
   ],
   resources: [
     {
-      href: 'OPS/subdir/1902/toc.xhtml',
-      type: 'application/xhtml+xml',
-      id: 'toc',
-      rel: 'contents',
+      type: 'application/vnd.ms-opentype',
+      href: 'OPS/subdir/1902/fonts/STIXGeneral.otf',
```

Comment:
Interesting, do you know why this was needed? Or rather why it wasn't needed previously?

Reply:
We had the TOC resource previously here(toc.xhtml was labeled linear="no"), but since we now add the TOC resource to the readingOrder, TOC no longer exists in the resources array so I just added this one for testing.

Reply:
Oh I see you added it in the OPF above. Sounds good!